dbt/models/default/columns.md

```diff
@@ -0,0 +1,384 @@
+## permit_address_full
+
+{% docs column_permit_address_full %}
+Address where permitted work will take place.
+
+This field combines the "mailing address" field in the source data
+with the concatenated output of the address component fields like
+`address_street_dir`, `address_street_name`, etc. We prefer the
+mailing address if it exists, and otherwise we substitute the
+concatenated address components.
+
+This field usually includes the municipality name for municipalities
+outside Chicago. If the municipality name is missing, the address
+is either in Chicago, or we generated it based on the concatenation
+of the address component fields, which do not include municipality
+name.
+{% enddocs %}
+
+## permit_assessable
+
+{% docs column_permit_assessable %}
+Whether the permitted work will affect the property's assessed
+values.
+
+This field is set by CCAO permit specialists who review the permit's
+work description to determine assessability.
+
+Possible values for this variable are:
+
+- null: The permit has not been reviewed yet
+- `A`: Assessable
+- `B`: Non-Assessable
```

Comment:
I think it makes sense in cases where there's a semantic value other than "this record is missing". I would definitely keep the null documentation here, since I wouldn't have guessed that to be the meaning.

Reply:
Sounds good! Note that again, I'm not totally confident that this is the meaning of the null value, but I'll make sure to clear it up on Thursday.